app/javascript/controllers/removals_controller.js

```diff
@@ -0,0 +1,27 @@
+import {Controller} from "@hotwired/stimulus"
+
+class RemovalsController extends Controller {
```

Comment:
What exactly are you using this for?

Reply:
It's a controller I've adapted from a previous project; it is used to remove the `flash` after its animation!